src/main/java/com/exasol/adapter/capabilities/MainCapability.java

```diff
@@ -87,5 +87,57 @@
      *
      * <p>Example: SELECT * FROM t LIMIT 100 OFFSET 10;</p>
```

Comment:
Use `<code>...</code>` around examples. Drop the trailing `;`

Reply:
done